### PR TITLE
Configure ci

### PR DESCRIPTION
**Pull request by Shrikant1212**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -86,39 +84,101 @@ jobs:
...
+          name: Commit into git
+          command: |
+            git config credential.helper 'cache --timeout=120'
+            git config user.email "shrikant1997upadhyay@gmail.com"
+            git config user.name "Shrikant Upadhyay"
+            git add .
+            git commit  -m "Commit from CircleCI"
+            git push -q https://$GIT_TOKEN@github.com/Shrikant1212/contracts.git
 
+  publish-package:
```
Comment (SkandaBhat) [inline .circleci/config.yml:166]: Cannot see this job being called anywhere.

**Review by SkandaBhat — CHANGES_REQUESTED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -86,39 +84,101 @@ jobs:
...
+    docker:
+      - image: "cimg/node:16.5"
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          name: Commit into git
+          command: |
+            git config credential.helper 'cache --timeout=120'
+            git config user.email "shrikant1997upadhyay@gmail.com"
```
[the remark this answers is not in the document]
Comment (adi44) [reply .circleci/config.yml:160]: Yes I have created one for gascompare

**Comment by SkandaBhat** [comment]
@Shrikant1212 any updates here?

**Comment by SkandaBhat** [comment]
Closing this. Please open when ready to merge @Shrikant1212

**Pull request closed without merging**